runtime/caml/compatibility.h

```diff
@@ -35,4 +35,12 @@
 
 #define HAS_SOCKLEN_T 1
 
+#ifdef HAVE_WORKING_FMA
```

Comment:
I don't expect this commit's going to make it, but just as a note that I think these renamings are a little bit of churn too far... it might be different if we were making a wholesale "change them all" (but please don't do that!!)